br/pkg/restore/db_test.go

```diff
@@ -425,5 +425,5 @@ func TestGetExistedUserDBs(t *testing.T) {
 //
 // The above variables are in the file br/pkg/restore/systable_restore.go
 func TestMonitorTheSystemTableIncremental(t *testing.T) {
-	require.Equal(t, int64(185), session.CurrentBootstrapVersion)
+	require.Equal(t, int64(186), session.CurrentBootstrapVersion)
```

Comment:
/cc @Leavrth please check if BR needs to make more change

Reply:
Is there any risk to directly update `last_analyze_version` with `restoreTS`?
https://github.com/pingcap/tidb/blob/dd1c635a8bc22a4220f402b7401f62e0cad4c553/br/pkg/restore/db.go#L141-L160